### ISSUE TITLE
Some alerts are not read in Microsoft Edge with the latest alpha build of NVDA

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#13615]
### Steps to reproduce:
1. Install the latest alpha build of NVDA (alpha-25209,21367c8e).
2. Launch Microsoft Edge.
3. Invoke Edge's menu -> Help and Feedback -> About Microsoft Edge.
4. Observe what happens when the "About Microsoft Edge" page is loaded.

### Actual behavior:
NVDA plays its error tone.

### Expected behavior:
NVDA should report "Microsoft Edge is up-to-date".

### System configuration
- Operating system: Windows 11 Pro version 21H2 (build 22000.613), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-25209,21367c8e, installed, in bulgarian.
- Microsoft Edge version: 100.0.1185.44, 64-bit, in bulgarian.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

#### Does the issue still occur after restarting your computer?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - 2021.3.5 - the problem does not occur with it.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, it is.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Unrelated, since the problem occurs with one version of NVDA but not with another on the same system.

### Notes
The "About Microsoft Edge" page is only an example. The same occurs at least in Google Meet during a call and in Google Docs while editing a document (the document's content is not read as it should be).

**Comment by seanbudd** [comment]
Is this an issue with 2022.1beta4? Could you provide a debug log with the error when using the latest alpha? A new alpha with a patch that may fix this is currently being built.

**Comment by k-kolev1985** [comment]
The issue is reproducible with NVDA 2022.1 beta 3.

I'm attaching a log with the issue reproduced with NVDA alpha-25209,21367c8e.
[NVDA_Log_2022-04-19_#13615.log](https://github.com/nvaccess/nvda/files/8511405/NVDA_Log_2022-04-19_.13615.log)

**Comment by seanbudd** [comment]
Can you test [alpha-25262,a55f9a62](https://ci.appveyor.com/project/nvaccess/nvda/build/alpha-25262,a55f9a62)? I am unable to reproduce this with both 2022.1beta3 and alpha-25209,21367c8e, using the same version of edge.

**Comment by k-kolev1985** [comment]
Strangely enough, I can still reproduce it with NVDA alpha-25262,a55f9a62. Should I provide more logs? Another page on which this is reproducible is speedtest.net - there should be some alerts about the test results there during and after a test, but there are none.

**Comment by seanbudd** [comment]
> Strangely enough, I can still reproduce it with NVDA alpha-25262,a55f9a62. Should I provide more logs? Another page on which this is reproducible is speedtest.net - there should be some alerts about the test results there during and after a test, but there are none.

With the log I can confirm that the error is the same as #13600 fixed by #13601. I believe updating to the latest alpha and restarting the system should fix this. If not, please upload a new log with `NVDA alpha-25262,a55f9a62` or newer

**Comment by k-kolev1985** [comment]
OK, I can still reproduce it with the latest NVDA alpha-25262,a55f9a62, even after restarting my system.

I'm providing a log created while reproducing the issue. And just in case, the logging was done with NVDA add-ons disabled.
[NVDA_Log_2022-04-22_#13615.log](https://github.com/nvaccess/nvda/files/8538029/NVDA_Log_2022-04-22_.13615.log)

**Comment by k-kolev1985** [comment]
An update on the issue. It seams that NVDA 2022.1 stable release is not affected - I have a copy of it installed on another machine and the issue does not occur with it. Also, the issue does not occur on my machine with a portable copy of NVDA 2022.1. So, there is something wrong, but it is in the alpha builds of NVDA.

**Comment by seanbudd** [comment]
It is difficult to understand the problem as in the log, all the relevant text from Edge is in Bulgarian.
The problem may also be related to the locale you are using.

Can you try providing a log and reproducing this using NVDA and Edge set to English? If possible, also using an English copy of Windows?

**Comment by k-kolev1985** [comment]
Here is a log with NVDA and Edge set to english. At the moment I don't have enough time to set my machine also to english, but I think I can do that on the weekend.

The problem is still reproducible.
[NVDA_Log_2022-07-12_#13615.log](https://github.com/nvaccess/nvda/files/9090146/NVDA_Log_2022-07-12_.13615.log)
